storybook/src/elements/nuxeo-html-editor/nuxeo-html-editor.stories.js

```diff
@@ -2,9 +2,9 @@ import { storiesOf } from '@storybook/polymer';
 import { html } from 'lit-html';
 import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-html-editor';
 
-storiesOf('UI/nuxeo-html-editor', module).addElement(
+storiesOf('UI/nuxeo-html-editor', module).add(
   'nuxeo-html-editor',
   () => html`
-    <nuxeo-html-editor></nuxeo-html-editor>
+    <nuxeo-html-editor> </nuxeo-html-editor>
```

Comment:
Is this spacing passing on linting/prettier? 🤔

Reply:
yeah, but fixed.